website/themes/uppy/layout/partials/sidebar.ejs

```diff
@@ -1,3 +1,23 @@
+<%
+var categories = [
+  // order: 1x
+  { path: 'docs/plugins/', name: 'Plugins', link: true },
+  // order: 2x
+  { path: 'docs/dashboard/', name: 'Local Sources', link: false },
+  // order: 3x
+  { path: 'docs/providers/', name: 'Remote Providers', link: true },
+  // order: 4x
+  { path: 'docs/tus/', name: 'Uploaders', link: false },
+  // order: 5x
+  { path: 'docs/status-bar/', name: 'UI Elements', link: false },
+  // order: 6x
+  { path: 'docs/transloadit/', name: 'File Processing', link: false },
+  // order: 7x
+  { path: 'docs/form/', name: 'Miscellaneous', link: false },
+  // order: 8x
+  { path: 'docs/react/', name: 'React Components', link: true },
```

Comment:
Reserved 10 items for each section … should maybe extract this to a json or yaml file like doc-order.json is.